core/src/main/java/org/elasticsearch/index/query/InnerHitBuilder.java

```diff
@@ -64,6 +59,7 @@
     public static final ParseField NAME_FIELD = new ParseField("name");
     public static final ParseField NESTED_PATH_FIELD = new ParseField("path");
     public static final ParseField PARENT_CHILD_TYPE_FIELD = new ParseField("type");
```

Comment:
are the NESTED_PATH_FIELD and PARENT_CHILD_TYPE_FIELD still useful? I would have expected them to be useless now that the top level syntax is gone?